pkg/libcose/patches/0001-RIOT-Use-RIOT-random_bytes-function-instead-of-rando.patch

```diff
@@ -1,94 +1,87 @@
-From 0104f8d1558fbd40d25dc48291508b3eac9965fa Mon Sep 17 00:00:00 2001
-From: Kaspar Schleiser <kaspar@schleiser.de>
-Date: Fri, 5 Jul 2019 14:51:09 +0200
-Subject: [PATCH] RIOT: Use RIOT random_bytes function instead of randombytes
+From d03612daa91c07448f097bfd40cf358771e0ddbe Mon Sep 17 00:00:00 2001
+From: Francisco Molina <femolina@uc.cl>
+Date: Thu, 24 Feb 2022 16:47:38 +0100
+Subject: [PATCH 1/3] RIOT: Use RIOT random_bytes function instead of
+ randombytes
 
 ---
- src/crypt/c25519.c     | 5 ++---
- src/crypt/hacl.c       | 6 +++---
- src/crypt/monocypher.c | 7 +++----
- 3 files changed, 8 insertions(+), 10 deletions(-)
+ src/cose_crypto.c      | 10 +++++++++-
+ src/crypt/c25519.c     |  4 ++--
+ src/crypt/hacl.c       |  1 -
+ src/crypt/monocypher.c |  4 ++--
+ 4 files changed, 13 insertions(+), 6 deletions(-)
 
+diff --git a/src/cose_crypto.c b/src/cose_crypto.c
+index 4dfd851..04b7e5a 100644
+--- a/src/cose_crypto.c
++++ b/src/cose_crypto.c
+@@ -11,8 +11,16 @@
+ #include <stdint.h>
+ #include <stdlib.h>
+ #include <string.h>
++#include "random.h"
+ 
+-cose_crypt_rng cose_crypt_get_random = NULL;
++static int _riot_random_bytes(void* arg, unsigned char * buf, size_t len)
++{
++    (void)arg;
++    random_bytes((uint8_t*) buf, len);
++    return 1;
++}
++
++cose_crypt_rng cose_crypt_get_random = _riot_random_bytes;
```

Comment:
Why not drop the patch and call `cose_crypt_set_rng()` instead?

Reply:
Done, but now depends on a libcose update

Reply:
Droped